ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -42,43 +86,32 @@ def test_numpy_ptp(
         input_dtypes=input_dtypes,
     )
 
-
-# nanpercentile
+# nanquantile
 @handle_frontend_test(
-    fn_tree="numpy.nanpercentile",
-    dtype_values_axis=_statistical_dtype_values(function="nanpercentile"),
-    where=np_frontend_helpers.where(),
+    fn_tree="numpy.nanquantile",
+    dtype_values_axis=_statistical_dtype_values(function="nanquantile"),
     keep_dims=st.booleans(),
+    test_with_out=st.just(False),
 )
-def test_numpy_nanpercentile(
+def test_numpy_nanquantile(
     dtype_values_axis,
-    where,
     frontend,
+    backend_fw,
     test_flags,
     fn_tree,
-    backend_fw,
-    on_device,
     keep_dims,
 ):
     input_dtypes, values, axis = dtype_values_axis
     if isinstance(axis, tuple):
         axis = axis[0]
 
-    where, input_dtypes, test_flags = np_frontend_helpers.handle_where_and_array_bools(
-        where=where,
-        input_dtype=input_dtypes,
-        test_flags=test_flags,
-    )
-
     np_frontend_helpers.test_frontend_function(
-        a=values[0][0],
-        q=values[0][1],
+        a=values[0],
+        q=values[1],  # Using the second value which represents quantiles

```

Comment:
I thinks tests is failing due to values being a list with the single element, which makes `values[1]` invalid